changelog/fragments/add-muiltiarch.yaml

```
@@ -0,0 +1,13 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (Alpha) New optional validator to help out verify the bundle against criteria defined to configure the support for multiple architectures. More info: https://olm.operatorframework.io/docs/advanced-tasks/ship-operator-supporting-multiarch/.
```

What do you think of this wording? Since this is part of the release notes I felt like this keeps it a bit more succinct. 
```suggestion
      (Alpha) New optional validator to help verify if a bundle matches criteria for multiple architecture support. More info: https://olm.operatorframework.io/docs/advanced-tasks/ship-operator-supporting-multiarch/.
```